server/app.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const path = require("path");
+
+const app = express();
+
+const sharksList = [
+  "https://founded.media/hiring/photos/sharks/11261840124_dc9ac72bbe_b.jpg",
+  "https://founded.media/hiring/photos/sharks/513197047_2f861d56cb_b.jpg",
+  "https://founded.media/hiring/photos/sharks/2989909952_b59500107e_o.jpg",
+  "https://founded.media/hiring/photos/sharks/4107884442_3baf8985f2_b.jpg",
+  "https://founded.media/hiring/photos/sharks/3822452418_ffa66da89d_o.jpg",
+  "https://founded.media/hiring/photos/sharks/3800013954_20fea3a9c9_b.jpg",
+  "https://founded.media/hiring/photos/sharks/7109693941_250fc6b246_k.jpg",
+  "https://founded.media/hiring/photos/sharks/8592704407_75c3c7ff53_h.jpg",
+  "https://founded.media/hiring/photos/sharks/14730744390_cebc28aa86_k.jpg",
+  "https://founded.media/hiring/photos/sharks/4936728723_91da549b05_b.jpg",
+];
+const catsList = [
+  "https://founded.media/hiring/photos/cats/14157413946_fea785b4d6_k.jpg",
+  "https://founded.media/hiring/photos/cats/16175483119_bd7374d8a8_h.jpg",
+  "https://founded.media/hiring/photos/cats/13901304865_a444cf4d34_k.jpg",
+  "https://founded.media/hiring/photos/cats/8311701653_49ed80202c_k.jpg",
+  "https://founded.media/hiring/photos/cats/13336301695_3c06dd41cc_k.jpg",
+  "https://founded.media/hiring/photos/cats/38679744435_66279af67c_k.jpg",
+  "https://founded.media/hiring/photos/cats/6393395037_9cda69da1a_b.jpg",
+  "https://founded.media/hiring/photos/cats/6977309082_44102ddf51_b.jpg",
+  "https://founded.media/hiring/photos/cats/11477923503_bbdf86387d_b.jpg",
+  "https://founded.media/hiring/photos/cats/4481336172_7f464f180d_b.jpg",
+];
+
+function shuffleArray(array) {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+}
+
+app.use(express.static(path.join(__dirname, "../client/build")));
+
+app.get("/api/animals", (req, res) => {
+  const animals = req.query.animals.split(",");
+  setTimeout(() => {
```

Comment:
can you explain why you used a setTimeout here?

Reply:
Just to slow down the server call a little bit so that you can catch the screen loading in the front-end. No other particular reason.